src/Theme_Mod_Command.php

```diff
@@ -232,6 +232,9 @@ public function remove( $args = array(), $assoc_args = array() ) {
 	 *     # Set theme mod
 	 *     $ wp theme mod set background_color 000000
 	 *     Success: Theme mod background_color set to 000000.
+	 *
+	 * @param string[] $args
```

Comment:
Different types then for `activate()` above. We should probably define and stick to a consistent type hint for command methods.

Reply:
Done!